Add ability to choose from multiple fragments

jdhalbert opened the pull request:
If a snippet has more than one fragment, each fragment will be displayed as an option for insertion.

![2022-08-18 20 57 01](https://user-images.githubusercontent.com/18095389/185539810-2a996faf-7718-4364-bbb2-c6b007f019e4.jpg)

antonreshetov commented:
@jdhalbert you need run `lint:fix`

antonreshetov commented on src/extension.ts:
```diff
@@ -11,36 +11,48 @@ export function activate (context: vscode.ExtensionContext) {
-
             if (snippet) {
-              snippetContent = snippet.content[0].value
+              if (snippet.content.length == 1) {
```
use `===` instead `==`

antonreshetov commented on src/extension.ts:
```diff
@@ -11,36 +11,48 @@ export function activate (context: vscode.ExtensionContext) {
+              }`
+            })
+          }
+        }
```
need space after parenthesis

antonreshetov commented on src/extension.ts:
```diff
@@ -11,36 +11,48 @@ export function activate (context: vscode.ExtensionContext) {
+
+          for (let fragment of snippet.content) {
+            let fragmentLabel = ''
+            if (snippet.content.length > 1) {
```
need space before `if`

antonreshetov commented:
P.S. Before you implement something, you need to agree on it, it is possible that your work will be in vain

jdhalbert commented:
Linted and replaced the == with ===. I don't understand the additional spaces requested.

I implemented it for my personal use. Feel free to use it or not. Thanks.
